app/developers/routes.py

```diff
@@ -898,11 +898,11 @@ def collection_confirmation(submission_id: UUID) -> ResponseReturnValue:
 def ask_a_question(submission_id: UUID, question_id: UUID) -> ResponseReturnValue:
     submission_helper = SubmissionHelper.load(submission_id)
     question = submission_helper.get_question(question_id)
-    answer = submission_helper.get_answer_for_question(question.id)
 
     # this method should work as long as data types are a single field and may
     # need to be revised if we have compound data types
-    form = build_question_form(question)(question=answer.root if answer else None)
+    submission_context = submission_helper.expression_context
+    form = build_question_form(question)(data=submission_context)
```

Comment:
This probably wants a `todo:` about multi-value question types needing to think through this step - I'm guessing the `data=` might need to be done by the form based on the questions that are passed in so it can split up the default values if it needs to 

(although this should work for all of our primitive question types right now so lets not do ti now)